ndex2/client.py

```diff
@@ -934,6 +1298,19 @@ def update_network_group_permission(self, groupid, networkid, permission):
     def update_network_user_permission(self, userid, networkid, permission):
         """
         Updated network user permission
+	
+	.. code-block:: python
+	
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		userid = 'INSERT USER ID HERE'
+		networkid = 'INSERT NETWORK UUID HERE'
+		#permission can be changed to other things
+		my_ndex.update_network_user_permission(userid, networkid, permission='READ'
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client